### PR TITLE
Add julia readme

### PR DESCRIPTION
**Pull request by davidmclifton**
I added material for Julia parser production to the main README and added a README file with step-by-step similar to the Rust README in the Julia plugin folder. I also added Julia to the list of plugins in the README located in the plugins directory.

### CODE REVIEW

File: src/plugins/julia/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+> NOTE: `npm` comes pre-installed with [Node.js](https://nodejs.org/en/)
+
+### 1. Generate a project environment
+
+Use Julia to generate the project by invoking the package management capability, which you can do by hitting the ']' key while in the Julia REPL:
+
+```
+MacBook-Pro-4:Personal Development maestro$ julia
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:18]: Let's remove specific computer names ("MacBook-Pro-4:Personal Development maestro"), and just put an abstract command line input

File: src/plugins/julia/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  Generating  project demo:
+    demo/Project.toml
+    demo/src/demo.jl
+
+(@v1.7) pkg> activate demo
+  Activating project at `~/Development/Personal Development/demo`
+
+(demo) pkg> status
+     Project demo v0.1.0
+      Status `~/Development/Personal Development/demo/Project.toml` (empty project)
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:38]: Ditto: let's put an abstract path to the example

File: src/plugins/julia/README.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+ _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
+|__/                   |
+
+(@v1.7) pkg> generate demo
+  Generating  project demo:
+    demo/Project.toml
+    demo/src/demo.jl
+
+(@v1.7) pkg> activate demo
+  Activating project at `~/Development/Personal Development/demo`
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:34]: Ditto

File: src/plugins/julia/README.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+(demo) pkg> 
+```
+
+### 2. Add the DataStructures dependency
+
+```
+(demo) pkg> add DataStructures
+    Updating registry at `~/.julia/registries/General.toml`
+   Resolving package versions...
+    Updating `~/Development/Personal Development/demo/Project.toml`
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:49]: Ditto

File: src/plugins/julia/README.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  [864edb3b] + DataStructures v0.18.11
+    Updating `~/Development/Personal Development/demo/Manifest.toml`
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:51]: Ditto

File: src/plugins/julia/README.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+(demo) pkg> 
+```
+
+### 3. Create your new Julia file
+
+Exit the Julia REPL by hitting the backspace key to go back to Julia prompt and exit() to end the process, then create a new Julia file for your code
+
+```
+julia> exit()
+MacBook-Pro-4:Personal Development maestro$ touch demo/src/Demo.jl
```
Comment (DmitrySoshnikov) [inline src/plugins/julia/README.md:108]: Ditto

**Review by DmitrySoshnikov — COMMENTED** [review]
Looks good, couple of comments inline:

**Comment by davidmclifton** [comment]
Completed edits

**Comment by DmitrySoshnikov** [comment]
Looks good, thanks!

**Pull request merged**